scala-mode2.el

```diff
@@ -169,5 +170,11 @@ When started, runs `scala-mode-hook'.
                '("\\.\\(scala\\|sbt\\)\\'" . scala-mode))
   (modify-coding-system-alist 'file "\\.\\(scala\\|sbt\\)\\'" 'utf-8))
 
+(defvar scala--prettify-symbols-alist scala-mode-pretty-all-alist)
```

Comment:
@fommil I thought it was already an opt-in feature as prettify-symbols mode is not enabled by default.